# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/components/newtab/bin/render-activity-stream-html.js
# Commit: 7ec776d31a2c
# Full Hash: 7ec776d31a2c6e0ffc1e5ee311d63993317ea7ca
# Author: Mike Conley <mconley@mozilla.com>
# Date: 2020-04-17 21:46:50
# Regressor Bug: 1614502
# File Overlap Count: 1
# Description:
#   Bug 1614502 - Expose utility functions for rendering Activity Stream. r=Mardak
#   
#   These will be used by a later patch inside of a ChromeWorker using
#   ReactDOMServer.
#   
# ==============================================================================

diff -r 8bf3fe111c40 -r 7ec776d31a2c browser/components/newtab/bin/render-activity-stream-html.js
--- a/browser/components/newtab/bin/render-activity-stream-html.js	Fri Apr 17 15:31:06 2020 +0000
+++ b/browser/components/newtab/bin/render-activity-stream-html.js	Fri Apr 17 15:31:11 2020 +0000
@@ -38,6 +38,7 @@
     `${options.baseUrl}vendor/react-redux.js`,
     `${options.baseUrl}vendor/react-transition-group.js`,
     `${options.baseUrl}data/content/activity-stream.bundle.js`,
+    `${options.baseUrl}data/content/newtab-render.js`,
   ];
 
   // Add spacing and script tags